imap_processing/cli.py

```diff
@@ -213,52 +332,52 @@ def process(self):
         """Perform SWE specific processing."""
         # self.file_path example:
         # imap/swe/l1a/2023/09/imap_swe_l1a_sci_20230927_20230927_v01-00.cdf
+        dependencies = self.download_dependencies()
         print(f"Processing SWE {self.level}")
+
+        # TODO: currently assumes just the first path returned is the one to use
```

Comment:
Yes. I can update this afterwards.

Reply:
Cool, I figured I could leave more of the SWE stuff for you to finalize, but I just didn't want to break it in my PR 😆